CMakeLists.txt

```diff
@@ -328,6 +328,11 @@ if (SINGULARITY_BUILD_TESTS)
 endif()
 # create the libsingularity-eos.a library
 if(SINGULARITY_USE_FORTRAN)
+  if (NOT SINGULARITY_BUILD_CLOSURE)
+    message(FATAL_ERROR "The fortran library relies on closures. "
```

Comment:
I'd say if we get here to just turn on build closure and warn that we are doing so.

Reply:
I'd rather be explicit. This came up because I wanted to build `compare_to_eospac` for GPU. But I didn't want to bother linking in kokkos kernels. So I want to get an error message that tells me what to use for that scenario.

Reply:
I'm willing to change it though if you feel strongly.

Reply:
It's worth thinking about in the future of not requiring any linear algebra library if we're not building closures, but for now I'll buy it.

Reply:
:+1: